src/sflow/hsflowd/patch/dropmon/0001-sflow-enabled-drop-monitor-support-for-SONiC.patch

```diff
@@ -1,49 +1,32 @@
-From 4bdd892662c08a396066ba6a1c55eac3f8aa0a5f Mon Sep 17 00:00:00 2001
```

Comment:
Can you please retain the original contributor details in the patch?

Reply:
Done.